packages/clerk-js/src/ui/common/InitialIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Text } from '../customizables';
+import { common } from '../styledSystem';
+
+type InitialIconProps = {
+  initials: string;
+};
+
+export const InitialIcon = ({ initials }: InitialIconProps) => {
+  return (
+    <Box
+      as='span'
+      sx={t => ({
+        ...common.centeredFlex('inline-flex'),
+        width: t.space.$4,
+        height: t.space.$4,
+        borderRadius: t.radii.$sm,
+        color: t.colors.$colorTextOnPrimaryBackground,
+        backgroundColor: t.colors.$primary500,
+      })}
+    >
+      <Text
+        as='span'
+        sx={t => ({
+          ...common.centeredFlex('inline-flex'),
+          width: '100%',
+          fontSize: t.fontSizes.$sm,
+          fontWeight: t.fontWeights.$semibold,
+        })}
+      >
+        {initials}
+      </Text>
+    </Box>
+  );
+};
```

Comment:
I believe we should either throw if the length of `initials` is more than one, or the logic of extracting the first letter from the string to be inside the component. I prefer the later